{"tag": "record", "value": {"event": "started", "filename": "screen-5200b76a4e77b43b.mp4", "orientation": "horizontal", "resolution": [1280, 720]}, "ts": 1534984717.8731499}
{"tag": "runnerRuntimeLog", "value": {"content": "[aircv][sift] threshold=0.6, result={'result': (564, 348), 'rectangle': [(0, 0), (0, 719), (1279, 719), (1279, 0)], 'confidence': 0.5032065499108285}"}, "ts": 1534984744.2293682}
{"tag": "runnerRuntimeLog", "value": {"content": "[aircv][sift] threshold=0.6, result={'result': (564, 348), 'rectangle': [(0, 0), (0, 719), (1279, 719), (1279, 0)], 'confidence': 0.5031980162020773}"}, "ts": 1534984745.7609797}
{"tag": "assertion", "value": {"method": "assert", "type": "True", "args": [false], "msg": "Failed to enter build screen", "lineno": null, "srcfilename": null}, "ts": 1534984745.7649682}
{"tag": "assertion", "value": {"method": "fail", "errmsg": "False is not true : Failed to enter build screen", "msg": "False is not true : Failed to enter build screen", "traceback": "Traceback: (failure cause)\n  File \"BuildMenu.py\", line 95, in <module>\n    pocounit.main()\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pocounit\\__init__.py\", line 45, in main\n    result = runner.run(suite)\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\unittest\\runner.py\", line 176, in run\n    test(result)\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\unittest\\suite.py\", line 84, in __call__\n    return self.run(*args, **kwds)\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pocounit\\suite.py\", line 15, in run\n    super(PocoTestSuite, self).run(result, debug)\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\unittest\\suite.py\", line 122, in run\n    test(result)\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\unittest\\case.py\", line 653, in __call__\n    return self.run(*args, **kwds)\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pocounit\\case.py\", line 201, in run\n    ret = self._super_run_modified(result)\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pocounit\\case.py\", line 262, in _super_run_modified\n    testMethod()\n  File \"BuildMenu.py\", line 84, in runTest\n    \"Failed to enter build screen\")\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pocounit\\case.py\", line 382, in assertTrue\n    self.fail(msg)\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pocounit\\case.py\", line 371, in fail\n    .format(''.join(traceback.format_stack()), err_typename, fail_msg))\nAssertionError: False is not true : Failed to enter build screen", "lineno": null, "srcfilename": null}, "ts": 1534984745.7719495}
{"tag": "siteSnapshot", "value": {"type": "screen", "dataPath": "snapshots/screen-2c9a8003b532ee007e7d1101103435c0.jpg", "site_id": "testcase_err_None", "width": 1280, "height": 720, "format": "jpg"}, "ts": 1534984746.1005173}
{"tag": "siteSnapshot", "value": {"type": "hierarchy", "dataPath": "snapshots/hierarchy-2c9a8003b532ee007e7d1101103435c0.json", "site_id": "testcase_err_None"}, "ts": 1534984746.1847744}
{"tag": "assertion", "value": {"method": "traceback", "errtype": "AssertionError", "traceback": "Traceback (most recent call last):\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pocounit\\utils\\outcome.py\", line 30, in testPartExecutor\n    yield\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pocounit\\case.py\", line 262, in _super_run_modified\n    testMethod()\n  File \"BuildMenu.py\", line 84, in runTest\n    \"Failed to enter build screen\")\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pocounit\\case.py\", line 382, in assertTrue\n    self.fail(msg)\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\pocounit\\case.py\", line 372, in fail\n    raise super(PocoTestCase, self).fail(fail_msg)\n  File \"C:\\Users\\Owner\\AppData\\Local\\Programs\\Python\\Python36\\lib\\unittest\\case.py\", line 670, in fail\n    raise self.failureException(msg)\nAssertionError: False is not true : Failed to enter build screen\n", "lineno": null, "srcfilename": null}, "ts": 1534984746.1877663}
{"tag": "assertion", "value": {"method": "assert", "type": "Equal", "args": [[], []], "msg": null, "lineno": null, "srcfilename": null}, "ts": 1534984749.8137717}
{"tag": "record", "value": {"event": "stopped", "filename": "screen-5200b76a4e77b43b.mp4"}, "ts": 1534984749.8157628}
{"tag": "siteSnapshot", "value": {"type": "screen", "dataPath": "snapshots/screen-1f281a662f5d03ac903c2dfedd579904.jpg", "site_id": "caseEnd", "width": 720, "height": 1280, "format": "jpg"}, "ts": 1534984754.0080676}
{"tag": "siteSnapshot", "value": {"type": "hierarchy", "dataPath": "snapshots/hierarchy-1f281a662f5d03ac903c2dfedd579904.json", "site_id": "caseEnd"}, "ts": 1534984754.1696353}
